### PR TITLE
fix: only handles IAM PR with required approval

### PR DESCRIPTION
**Pull request by sqin2019**
Fixes 57, to use this change, aod-template needs to add job permission `pull-requests: 'read'`

### CODE REVIEW

**Comment by sqin2019** [comment]
Hold until we explore the alternative flow.

File: .github/workflows/handle.yml [hunk 1]
```diff
@@ -43,9 +42,38 @@ env:
...
 jobs:
+  review_status:
+    runs-on: 'ubuntu-latest'
+    outputs:
+      REVIEW_DECISION: '${{ steps.get_review_decision.outputs.REVIEW_DECISION }}'
+    steps:
+      - id: 'repo_name'
+        env:
+          REPO: '${{ github.repository }}'
+        run: 'echo "REPO_NAME=${REPO##*/}" >> $GITHUB_OUTPUT'
```
Comment (yolocs) [inline .github/workflows/handle.yml:53]: Can we inline this in this next step?

File: .github/workflows/handle.yml [hunk 2]
```diff
@@ -43,9 +42,38 @@ env:
...
+      - id: 'get_review_decision'
+        env:
+          OWNER: '${{ github.repository_owner }}'
+          REPO_NAME: '${{ steps.repo_name.outputs.REPO_NAME }}'
+          PR_NUMBER: '${{ github.event.pull_request.number }}'
```
Comment (yolocs) [inline .github/workflows/handle.yml:58]: Consider inline the owner and PR number

File: .github/workflows/handle.yml [hunk 3]
```diff
@@ -43,9 +42,38 @@ env:
...
+          GH_TOKEN: '${{ github.token }}'
```
Comment (yolocs) [inline .github/workflows/handle.yml:59]: If this is needed, add a comment explain why and hopefully reference.

**Review by yolocs — COMMENTED** [review]
nits

**Review by yolocs — APPROVED** [review]

**Pull request merged**